tailwind.config.js

```diff
@@ -11,7 +11,11 @@ module.exports = {
       'serif': [...defaultTheme.fontFamily.serif],
       'mono': [...defaultTheme.fontFamily.mono],
     },
-    extend: {},
+    extend: {
```

Comment:
Do we need this?

Reply:
nope not yet, left behind after an experiment - thanks